CRM/Odoosync/Sync/Contribution/PendingContribution.php

```diff
@@ -47,6 +58,7 @@ public function getIds() {
         'is_deleted' => ['IS NOT NULL' => 1],
         'options' => ['limit' => (int) CRM_Odoosync_Sync_BatchSize::getCurrentBatchSize()],
         'custom_' . $this->syncStatusFieldId => $this->syncStatusValue,
+        'custom_' . $this->doNotSyncFieldId =>  ['!=' => 1]
```

Comment:
use TRUE instead of 1, also what if the contribution does not have that field set ? maybe you should also check if the value either != TRUE of it = NULL